core/store/migrate/migrations/0104_create_forwarder_addresses.sql

```diff
@@ -0,0 +1,14 @@
+-- +goose Up
+CREATE TABLE evm_forwarders (
+    id BIGSERIAL PRIMARY KEY,
+    address bytea NOT NULL UNIQUE,
+    created_at timestamptz NOT NULL,
```

Comment:
BRIN on created_at/updated_at is always useful

Reply:
done

Reply:
I'm not sure the usecase of having this on update for forwarders for now but the overhead is minimal anyway